### PR TITLE
Feature/pfm 21 create groups

### PR DESCRIPTION
**Pull request by Luisella21**
Create groups.
Add permission checks where needed.
Create tests.

### CODE REVIEW

File: peoplefinder/management/commands/create_people_finder_groups.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth.models import Group, Permission
+from django.core.management.base import BaseCommand
+
+
+PERSON_EDITOR_PERMISSION_TYPES = [
+    "edit_profile",
+    "delete_profile",
+    "edit_sso_id",
```
Comment (rossmiller) [inline peoplefinder/management/commands/create_people_finder_groups.py:8]: Please remove edit_sso_id.

File: peoplefinder/migrations/0057_auto_20220207_1206.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('peoplefinder', '0056_create_people_finder_groups'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='person',
+            options={'permissions': (('edit_profile', 'Edit profile'), ('delete_profile', 'Delete profile'), ('edit_sso_id', 'Edit SSO ID'))},
```
Comment (rossmiller) [inline peoplefinder/migrations/0057_auto_20220207_1206.py:15]: Remove edit_sso_id

File: peoplefinder/models.py [hunk 3]
```diff
@@ -202,6 +202,11 @@ class Meta:
                 check=~Q(pk=F("manager")), name="manager_cannot_be_self"
             ),
         ]
+        permissions = (
+            ("edit_profile", "Edit profile"),
+            ("delete_profile", "Delete profile"),
+            ("edit_sso_id", "Edit SSO ID"),
```
Comment (rossmiller) [inline peoplefinder/models.py:208]: Remove edit_sso_id

File: pii-ner-exclude.txt [hunk 4]
```diff
@@ -446,3 +446,8 @@ el = e.target
 e.preventDefault
 currentTeamId(team_id
 s3chunkuploader.fields
+edit_sso_id"
```
Comment (rossmiller) [inline pii-ner-exclude.txt:449]: Remove edit_sso_id"

File: peoplefinder/test/views/test_profile_views.py [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+    response = state.client.get(test_url)
+    assert response.status_code == 200
+    assert button_title in response.content
+    soup = BeautifulSoup(response.content, features="html.parser")
+    buttons = soup.find_all('a')
+    assert len(buttons) == button_len + 1
+
+def check_permission(state, view_url, codename):
+
```
Comment (rossmiller) [inline peoplefinder/test/views/test_profile_views.py:65]: Remove line break

**Review by rossmiller — CHANGES_REQUESTED** [review]
Looks good but we now need to remove the edit_sso_id

File: peoplefinder/test/views/test_profile_views.py (hunk 5 above)
Comment (Luisella21) [reply peoplefinder/test/views/test_profile_views.py:64]: Done.

**Review by rossmiller — APPROVED** [review]
LGTM!

**Pull request merged**